docs/threat_model/threat_model.rst

```diff
@@ -473,7 +473,6 @@ What a Compromise of the Journalist's Property Can Achieve
    -  Access the Hidden Service values used by the *Journalist Interface*.
    -  Access SSH keys and passphrases for the *Application Server* and the
       *Monitor Server*.
-   -  Access the journalist's personal GPG key.
```

Comment:
Threat model doc refers to SD 0.3 - unclear if it should be changed piecemeal, or edited in its entirety to include references to other assets. This change adds an Export USB and recommends the use of a password manager on the journalist's mobile device, neither of which are covered here.

Reply:
Per discussion earlier today, I've made minimal updates to the threat model consistent with the new terminology and updated data flow in 60cc802531e97019bdc76c1723a0dde6eff66494.